config-overrides.js

```diff
@@ -0,0 +1,6 @@
+module.exports = function override(config, env) {
```

Comment:
Might be worth adding a comment to this file to explain it is used by react-app-rewired.

It would be self-explanatory if this file was called "react-app-rewired-config.js"... oh well... 🙄

Reply:
Yep, I've looked around the react-app-rewired and there's no way to use another filename. If you want we can use another one to do the job (I've saw that proxyquire does a similar job, but needs some setup)